### PR TITLE
update typecast without build script

### PR DESCRIPTION
**Pull request by rouzwelt**
## Motivation
related to https://github.com/rainlanguage/alloy-ethers-typecast/pull/36
update the typecast version that has no build script

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [ ] unit-tested any new functionality
- [x] linked any relevant issues or PRs
- [ ] included screenshots (if this involves a front-end change)

### CODE REVIEW

File: tauri-app/src-tauri/Cargo.lock [hunk 1]
```diff
@@ -269,7 +269,7 @@ dependencies = [
 [[package]]
 name = "alloy-ethers-typecast"
 version = "0.2.0"
-source = "git+https://github.com/rainlanguage/alloy-ethers-typecast?rev=0881930a22e84db49ba955c5b88e790e1266ac66#0881930a22e84db49ba955c5b88e790e1266ac66"
+source = "git+https://github.com/rainlanguage/alloy-ethers-typecast?rev=0e636d2bc865a49c6eaea95ab139465708a65698#0e636d2bc865a49c6eaea95ab139465708a65698"
```
Comment (hardyjosh) [inline tauri-app/src-tauri/Cargo.lock:272]: Old commit

File: tauri-app/src-tauri/Cargo.lock [hunk 2]
```diff
@@ -8109,10 +8109,10 @@ checksum = "dc33ff2d4973d518d823d61aa239014831e521c75da58e3df4840d3f47749d09"
 [[package]]
 name = "rain-erc"
 version = "0.0.0"
-source = "git+https://github.com/rainlanguage/rain.erc?rev=80bb411b161c1ee992a7974d325a783214046d5b#80bb411b161c1ee992a7974d325a783214046d5b"
+source = "git+https://github.com/rainlanguage/rain.erc?rev=e86b46bfce535f4c562b93a212171fbf19953a8f#e86b46bfce535f4c562b93a212171fbf19953a8f"
 dependencies = [
  "alloy",
- "alloy-ethers-typecast 0.2.0 (git+https://github.com/rainlanguage/alloy-ethers-typecast?rev=0881930a22e84db49ba955c5b88e790e1266ac66)",
+ "alloy-ethers-typecast 0.2.0 (git+https://github.com/rainlanguage/alloy-ethers-typecast?rev=0e636d2bc865a49c6eaea95ab139465708a65698)",
```
Comment (hardyjosh) [inline tauri-app/src-tauri/Cargo.lock:8115]: Old commit

File: tauri-app/src-tauri/Cargo.lock [hunk 3]
```diff
@@ -8169,7 +8169,7 @@ name = "rain-metadata"
 version = "0.0.2-alpha.6"
 dependencies = [
  "alloy",
- "alloy-ethers-typecast 0.2.0 (git+https://github.com/rainlanguage/alloy-ethers-typecast?rev=0881930a22e84db49ba955c5b88e790e1266ac66)",
+ "alloy-ethers-typecast 0.2.0 (git+https://github.com/rainlanguage/alloy-ethers-typecast?rev=0e636d2bc865a49c6eaea95ab139465708a65698)",
```
Comment (hardyjosh) [inline tauri-app/src-tauri/Cargo.lock:8172]: Old commit

**Review by hardyjosh — CHANGES_REQUESTED** [review]
Lockfile still shows uses of the old version

File: tauri-app/src-tauri/Cargo.lock (hunk 3 above)
Comment (rouzwelt) [reply tauri-app/src-tauri/Cargo.lock]: fixed

File: tauri-app/src-tauri/Cargo.lock (hunk 1 above)
Comment (rouzwelt) [reply tauri-app/src-tauri/Cargo.lock]: fixed

**Review by hardyjosh — APPROVED** [review]

**Pull request merged**